src/main/resources/db/h2/bootstrap_data.sql

```diff
@@ -22,4 +22,20 @@ VALUES
 (8, 2, 'title', 'Introduction to Programming'),
 
 (9, 3, 'professorName', 'Curtis D''Alves'),
-(10, 3, 'title', 'Practice and Expertise');
\ No newline at end of file
+(10, 3, 'title', 'Practice and Expertise');
+
+INSERT INTO ASSIGNMENT (ID, COURSE_ID, NAME, DUE_DATE)
+VALUES
+(1, 1, 'Haskell Basics')
+(2, 2, 'Discrete math')
```

Comment:
Does this work when you run the app? I think you're missing some syntax. Same for below.
```suggestion
INSERT INTO ASSIGNMENT (ID, COURSE_ID, NAME, DUE_DATE)
VALUES
(1, 1, 'Haskell Basics'),
(2, 2, 'Discrete math');
```